src/Umbraco.Core/Sync/IBatchedDatabaseServerMessenger.cs

```diff
@@ -0,0 +1,7 @@
+namespace Umbraco.Core.Sync
+{
+    public interface IBatchedDatabaseServerMessenger : IServerMessenger
```

Comment:
Code comments? need to explain what this is ... though this might have been empty before, if so not to worry